wrapper/src/main/java/software/amazon/jdbc/plugin/customendpoint/CustomEndpointMonitorImpl.java

```diff
@@ -168,12 +171,30 @@ public void monitor() {
           TimeUnit.NANOSECONDS.sleep(sleepDuration);
         } catch (InterruptedException e) {
           throw e;
+        } catch (RdsException ex) {
+          LOGGER.log(Level.SEVERE,
```

Comment:
should this still be SEVERE if it's a throttling exception? especially given the full stack trace